apps/console/src/features/server-configurations/pages/connector-listing-page.tsx

```diff
@@ -189,25 +191,26 @@ export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterfa
     };
 
     /**
-     * This function returns loading placeholder.
+     * This function returns loading placeholders.
      */
     const renderLoadingPlaceholder = (): ReactElement => {
-        return (
-            <Grid.Row columns={ 1 }>
-                <div>
+        const placeholders: ReactElement[] = [];
+        
+        for (let loadedPlaceholders: number = 0; loadedPlaceholders <= 1; loadedPlaceholders++) {
+            placeholders.push(
+                <Grid xs={ 12 } lg={ 6 } key={ loadedPlaceholders }>
                     <div 
                         className="ui card fluid settings-card"
                         data-testid={ `${testId}-loading-card` }
                     >
                         <div className="content no-padding">
-                            <div className="header-section">
+                            <div className="header-section" style={ { "paddingBottom": "50px" } }>
```

Comment:
shall we move this inline css to a seperate file ?

Reply:
fixed https://github.com/wso2/identity-apps/pull/4297/commits/e20e1633f26f50be611842ccc3f5b9cc3b49b142